components/eam/docs/user-guide/aerosol_phys_prop.md

```diff
@@ -0,0 +1,109 @@
+
+# Aerosol physical properties
+
+## Description
+
+Key information
+
+- Original physical properties of aerosols are documented in Liu et al. (2012). Detailed information is included in the supplement.
+
+- Physical properties of aerosols used in E3SMv1 are documented in Wang et al. (2020).
+
+- Marine organic aerosol properties are defined in Burrows et al. (2022).
+
+- Dust refractive index and longwave absorption coefficients are updated by Feng et al. (2022).
+
+- BC and POM hygroscopicity values are updated by Shan et al. (2024).
+
+- <span style="color:red">Some description/reference about the 5th mode.</span>
```

Comment:
@keziming - would you suggest a short description of the 5th mode here?

Reply:
Physical properties of the fifth mode aerosols are documented by Ke et al. (2024, in preparation)